packages/Meshfree/test/PointCloudProblemGenerator/ExodusProblemGenerator_def.hpp

```diff
@@ -351,7 +355,7 @@ void ExodusProblemGenerator<Scalar, SourceDevice, TargetDevice>::
     }
 
     // Build a communication plan for the sources.
-    Tpetra::Distributor distributor( _comm );
+    Tpetra::Distributor distributor( _comm ); // FIXME???
```

Comment:
Can we use your distributor here?

Reply:
For no good reason.  I thought you were using the `Teuchos::Comm` but it seems you only get the size and the rank...